src/redux/reducers/decisions.ts

```diff
@@ -0,0 +1,25 @@
+import { CORE_DECISIONS_LIST } from '~redux/constants';
+import { Decision } from '~redux/immutable/Decision';
+import { CoreDecisions, CoreDecisionsRecord } from '~redux/state/decisions';
+import { ReducerType } from '~redux/types';
+
+import { ActionTypes } from '../actionTypes';
+
+const coreDecisionsReducer: ReducerType<CoreDecisionsRecord> = (
+  state = CoreDecisions(),
+  action,
+) => {
+  switch (action.type) {
+    case ActionTypes.DECISION_DRAFT_CREATED:
+      return state.setIn(
```

Comment:
This will set one decision per wallet address to a list of draft decisions being held by the store.